test/unit/effects.js

```diff
@@ -1850,6 +1852,35 @@ test("Animation callbacks (#11797)", 15, function() {
 	this.clock.tick( 10 );
 });
 
+test("Animation callbacks (#2292)", 1, function() {
+	var jq_elem = jQuery("#foo"),
```

Comment:
It's because I'm used to add `jq_` for every jQuery object, but I removed the variable anyway, thanks.